snippets/serializeForm.md

```diff
@@ -0,0 +1,21 @@
+### serializeForm
+
+Serializes a form into an object.
+
+First we transform the `form` into `FormData`, then we convert it into an `array` and from the `array` we collect an `query string`
```

Comment:
Explain a bit more in-depth the methods used, see existing snippets for reference.